src/coprocessor/dag/rpn_expr/types/expr.rs

```diff
@@ -32,24 +32,17 @@ pub enum RpnExpressionNode {
     },
 
     /// Represents a reference to a column in the columns specified in evaluation.
-    ColumnRef {
-        offset: usize,
-
-        // Although we can know `ColumnInfo` according to `offset` and columns info in scan
-        // executors, its type is `ColumnInfo` instead of `FieldType`.
-        // Maybe we can remove this field in future.
-        field_type: FieldType,
-    },
+    ColumnRef { offset: usize },
```

Comment:
A note for reviewers: We now pass FieldType in schema, so we don't need to store field_type any more.